sandboxed_containers/deploying-sandboxed-container-workloads.adoc

```diff
@@ -14,6 +14,29 @@ include::modules/sandboxed-containers-preparing-openshift-cluster.adoc[leveloffs
 .Additional resources
 * xref:../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[Installing a user-provisioned cluster on bare metal]
 
+[id="checking-nodes-sandboxed-containers"]
+== Checking if you can deploy {sandboxed-containers-first} in your cluster
+
+Before running {sandboxed-containers-first} containers, you need to check whether the nodes in your cluster are eligible to run Kata containers. Some cluster nodes might not comply with sandboxed containers' minimum requirements. The most common reason for node ineligibility is the lack of virtualization support on the node. If you attempt to run sandboxed workloads on ineligible nodes, you will experience errors. You can use the Node Feature Discovery Operator (NFD) and a `NodeFeatureDiscovery` resource to automatically check node eligibility. When installed, NFD prevents installation of the sandboxed runtime on ineligible nodes.
+
+[NOTE]
+====
+If you want to install the Kata runtime on selected worker nodes only that you know are eligible, apply the `feature.node.kubernetes.io/runtime.kata=true` label to the selected nodes and set `checkNodeEligibility: false` in the `KataConfig` resource. See xref:../sandboxed_containers/deploying-sandboxed-container-workloads.adoc#sandboxed-containers-deploying-kata-runtime-select-nodes_deploying-sandboxed-containers[Deploying on selected nodes].
```

Comment:
"If you want to install the Kata runtime on selected worker nodes only that you know are eligible" > "If you want to install the Kata runtime only on selected worker nodes that you know are eligible"

Reply:
Interesting. This is either a blind spot of mine when it comes to the English language, or an interesting difference between Hiberno-English and American-English usage. I've had this exact comment before, by Ashley Hardin IIRC. 

I think both forms are probably OK? WDYT?